### PR TITLE
Update sky parameters for new ambient occlusion and ambient light

### PR DESCRIPTION
**Pull request by kamil-shapespark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ settings. If `sky` is missing, the default sky settings are used, if
 defaults to `6`.
 + `color`: optional, three RGB values in `[0,1]` range, in linear
 color space, defaults to `[0.855,0.863,1]`.
++ `ambientLight`: optional, an object that configures ambient
+lighting parameters. If `ambientLight` is missing, default ambient lighting
```
Comment (wmatyjewicz) [inline README.md]: lighting > light for consistency with the UI?

File: README.md [hunk 2]
```diff
@@ -204,10 +208,16 @@ ambient occlusion is disabled.
 sky texture that surrounds the scene. The texture is not used for
 baking.
 
+`ambientLight` has the following properties:
+
++ `strength` optional, a float that specifies how strong is the effect
+of ambient light, defaults to `0.05`. `0` is an equivalent of
```
Comment (wmatyjewicz) [inline README.md]: a float that specifies ambient light strength

File: README.md [hunk 3]
```diff
@@ -204,10 +208,16 @@ ambient occlusion is disabled.
...
+disabled ambient lighting.
+
 `ambientOcclusion` has the following properties:
 
-+ `factor` optional, a float that specifies how strong is the effect
-of ambient occlusion, defaults to `0.05`. `0` is an equivalent of
++ `intensity` optional, a float that specifies how strong is the effect
+of ambient occlusion, defaults to `0.5`. `0` is an equivalent of
```
Comment (wmatyjewicz) [inline README.md]: a float that specifies how strong the effect of ambient occlusion is (passive voice)

I'd drop the 0 is an equivalent...

File: README.md [hunk 4]
```diff
@@ -196,6 +196,10 @@ settings. If `sky` is missing, the default sky settings are used, if
 defaults to `6`.
 + `color`: optional, three RGB values in `[0,1]` range, in linear
 color space, defaults to `[0.855,0.863,1]`.
++ `ambientLight`: optional, an object that configures ambient
```
Comment (wmatyjewicz) [inline README.md]: ambientLight is now a separate object, right?

File: README.md [hunk 5]
```diff
@@ -196,6 +196,10 @@ settings. If `sky` is missing, the default sky settings are used, if
...
+lighting parameters. If `ambientLight` is missing, default ambient lighting
+parameters are used, if `ambientLight` is set to null, ambient lighting is
+disabled.
 + `ambientOcclusion`: optional, an object that configures ambient
```
Comment (wmatyjewicz) [inline README.md]: ambientOcclusion is a separate object now, right?

**Review by wmatyjewicz — CHANGES_REQUESTED** [review]

File: extras.json [hunk 6]
```diff
@@ -33,8 +33,11 @@
   "sky": {
     "strength": 6,
     "color": [0.854, 0.862, 1],
+    "ambientLight": {
```
Comment (wmatyjewicz) [inline extras.json]: ambientLight and ambientOcclusion are separate objects now.

Comment (wmatyjewicz) [inline extras.json]: The cubes.zip example has to be updated.

**Review by wmatyjewicz — CHANGES_REQUESTED** [review]

File: extras.json (hunk 6 above)
Comment (wmatyjewicz) [reply extras.json]: When you have new cubes.zip could you also update e2e_tests in the shapespark repo?

File: README.md (hunk 1 above)
Comment (kamil-shapespark) [reply README.md]: Yes, Light will be better

File: README.md (hunk 2 above)
Comment (kamil-shapespark) [reply README.md]: OK!

File: README.md (hunk 3 above)
Comment (kamil-shapespark) [reply README.md]: Good point, thanks

File: README.md (hunk 4 above)
Comment (kamil-shapespark) [reply README.md]: Yes, my bad, fixed

File: README.md (hunk 5 above)
Comment (kamil-shapespark) [reply README.md]: Yes, fixed that

File: extras.json (hunk 6 above)
Comment (kamil-shapespark) [reply extras.json]: Yes, fixed that, thanks

Comment (kamil-shapespark) [reply extras.json]: I've updated cubes.zip in this repo and in the `shapespark` repo.

File: README.md [hunk 7]
```diff
@@ -196,31 +196,40 @@ settings. If `sky` is missing, the default sky settings are used, if
...
 
 + `fileName`: required, name of the file inside the archive that
 stores the sky texture.
 + `yawRotation` optional number in `[0,360]` range that specifies the
 rotation of the sky texture in degrees. Defaults to `0`.
 
+### `ambientLight` object
+
+`ambientLight` is an optional object that configures ambient light parameters.
+If `ambientLight` is missing, default ambient light
```
Comment (wmatyjewicz) [inline README.md]: Since it's only one parameter, what about: "default ambient light is used".

File: README.md [hunk 8]
```diff
@@ -196,31 +196,40 @@ settings. If `sky` is missing, the default sky settings are used, if
...
+parameters are used, if `ambientLight` is set to null, ambient light is
+disabled.
+
++ `strength` optional, a float that specifies ambient light strength, 
+defaults to `0.05`. `0` is an equivalent of disabled ambient lighting.
```
Comment (wmatyjewicz) [inline README.md]: I'd drop the "0 is an equivalent..." part.

File: README.md [hunk 9]
```diff
@@ -196,31 +196,40 @@ settings. If `sky` is missing, the default sky settings are used, if
...
+
+### `ambientOcclusion` object
+
+`ambientOcclusion`: is an optional object that configures ambient
```
Comment (wmatyjewicz) [inline README.md]: no colon

File: README.md [hunk 10]
```diff
@@ -196,31 +196,40 @@ settings. If `sky` is missing, the default sky settings are used, if
...
 rotation of the sky texture in degrees. Defaults to `0`.
 
+### `ambientLight` object
+
+`ambientLight` is an optional object that configures ambient light parameters.
+If `ambientLight` is missing, default ambient light
+parameters are used, if `ambientLight` is set to null, ambient light is
+disabled.
+
++ `strength` optional, a float that specifies ambient light strength,
```
Comment (wmatyjewicz) [inline README.md:218]: The `ambientLight` object has the following properties:

File: README.md [hunk 11]
```diff
@@ -196,31 +196,40 @@ settings. If `sky` is missing, the default sky settings are used, if
...
+defaults to `0.05`. `0` is an equivalent of disabled ambient lighting.
+
+### `ambientOcclusion` object
+
+`ambientOcclusion`: is an optional object that configures ambient
+occlusion parameters. If `ambientOcclusion` is missing, default ambient
+occlusion parameters are used, if `ambientOcclusion` is set to null,
+ambient occlusion is disabled.
+
++ `intensity` optional, a float that specifies how strong the effect
```
Comment (wmatyjewicz) [inline README.md:230]: The `ambientOcclusion` object has the following properties:

**Review by wmatyjewicz — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (kmmod) [reply README.md]: Yes, that seems to be better

File: README.md (hunk 10 above)
Comment (kmmod) [reply README.md:218]: OK

File: README.md (hunk 8 above)
Comment (kmmod) [reply README.md]: OK

File: README.md (hunk 9 above)
Comment (kmmod) [reply README.md]: Thanks!

File: README.md (hunk 11 above)
Comment (kmmod) [reply README.md:230]: OK

**Review by wmatyjewicz — APPROVED** [review]

**Pull request merged**